tuna/miopen/driver/base.py

```diff
@@ -39,11 +39,12 @@
 from tuna.miopen.db.miopen_tables import ConvolutionConfig
 from tuna.miopen.utils.metadata import TENSOR_PRECISION
 from tuna.miopen.utils.parsing import parse_line
+from tuna.driver import DriverBase
 
 LOGGER = setup_logger('driver_base')
 
 
-class DriverBase():
+class MIOpenDriver(DriverBase):
```

Comment:
the __init__  here needs to be moved to the DriverBase class, exactly as it is. Then call super() here.